Documents/com~apple~CloudDocs/Queen/bin/Activate.ps1

```diff
@@ -0,0 +1,241 @@
+<#
+.Synopsis
+Activate a Python virtual environment for the current PowerShell session.
+
+.Description
+Pushes the python executable for a virtual environment to the front of the
+$Env:PATH environment variable and sets the prompt to signify that you are
+in a Python virtual environment. Makes use of the command line switches as
+well as the `pyvenv.cfg` file values present in the virtual environment.

```

Comment:
Al these files have been accidentally added into your git push. I think this might be related to using `git add .` to add all your files. These need to be deleted and in the future, add by file name or add this folder to your gitignore.